dashboard/app/models/peer_review.rb

```diff
@@ -0,0 +1,43 @@
+# == Schema Information
+#
+# Table name: peer_reviews
+#
+#  id              :integer          not null, primary key
+#  user_id         :integer
+#  from_instructor :boolean          default(FALSE), not null
+#  script_id       :integer          not null
+#  level_id        :integer          not null
+#  level_source_id :integer          not null
+#  data            :text(65535)
+#  status          :integer
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#
+# Indexes
+#
+#  index_peer_reviews_on_level_id         (level_id)
```

Comment:
Do we need the index on level_id? How often are we going to say "show me all peer reviews for this task?" as opposed to "show me all peer reviews for this module?"

We're probably not at the point where this table is overindexed but its still worth thinking about